README.md

```diff
@@ -663,7 +664,8 @@ def test_pebble_exec():
         ctx.on.pebble_ready(container),
         state_in,
     )
-    assert state_out.containers["foo"].get_exec(['ls', '-ll']).stdin == "..."
+    assert ctx.exec_history[container.name].command == ['ls', '-ll']
```

Comment:
Does there need to be a `[0]` here, if it's history? In other words, is this a list of execs?

Reply:
Ah, yes, I missed change that one. Thanks, will fix.